src/Common/src/System/Net/Http/HttpHandlerDefaults.cs

```diff
@@ -16,5 +16,6 @@ internal static class HttpHandlerDefaults
         public const bool DefaultPreAuthenticate = false;
         public const ClientCertificateOption DefaultClientCertificateOption = ClientCertificateOption.Manual;
         public const bool DefaultUseProxy = true;
+        public readonly static TimeSpan DefaultConnectTimeout = TimeSpan.FromSeconds(60);
```

Comment:
> We discussed this months ago and @weshaggard updated the coding-guidelines with the answer to this.

I did, actually :)
https://github.com/dotnet/corefx/commit/88c467b94aeed05c07fa14cc2769f49d66125ab3
based on the same reasoning.  I just didn't follow it here due to copy-pasting from existing code and not noticing it was the wrong order.